### PR TITLE
fixed #1944: The subclass of cc.Sprite,cc.Layer... doesn't need cc.associateWithNative anymore.

### PR DESCRIPTION
**Pull request by dumganhar**

### CODE REVIEW

**Comment by jdmunro** [comment]
Looks good - is it a good idea to commit commented out code?

**Comment by dumganhar** [comment]
@jdmunro 
Yes, I think i should remove the unused codes. Just keep an empty implementation.
Thank you.

File: scripting/javascript/bindings/js_bindings_opengl.cpp [hunk 1]
```diff
@@ -60,6 +60,18 @@ JSBool js_cocos2dx_GLNode_constructor(JSContext *cx, uint32_t argc, jsval *vp)
...
 }
 
+static JSBool js_cocos2dx_GLNode_ctor(JSContext *cx, uint32_t argc, jsval *vp)
+{
+	JSObject *obj = JS_THIS_OBJECT(cx, vp);
+    GLNode *nobj = new GLNode();
+    js_proxy_t* p;
+    JS_NEW_PROXY(p, nobj, obj);
+    nobj->autorelease();
+    JS_AddNamedObjectRoot(cx, &p->obj, "cocos2d::extension::CCBReader");
```
Comment (ricardoquesada) [inline scripting/javascript/bindings/js_bindings_opengl.cpp]: Probably it should not say `"cocos2d::extension:CCBReader"`... perhaps it should say something like `"GLNode"`

Comment (dumganhar) [inline scripting/javascript/bindings/js_bindings_opengl.cpp]: Sure, it's a `copy` `paste` mistake. I'll correct it. Thanks.

**Comment by dumganhar** [comment]
@ricardoquesada 
Updated.

**Pull request merged**